torch_geometric/utils/spmm.py

```diff
@@ -1,23 +1,23 @@
 import torch
 from torch import Tensor
 
-from torch_geometric.typing import Adj, SparseTensor, torch_sparse
-from torch_geometric.utils import is_torch_sparse_tensor
+from torch_geometric.typing import Adj, SparseTensor
 
 
 @torch.jit._overload
-def spmm(src, other, reduce):
-    # type: (Tensor, Tensor, str) -> Tensor
+def spmm(src, other, reduce, convert_to_csr):
+    # type: (SparseTensor, Tensor, str, bool) -> Tensor
     pass
 
 
 @torch.jit._overload
-def spmm(src, other, reduce):
-    # type: (SparseTensor, Tensor, str) -> Tensor
+def spmm(src, other, reduce, convert_to_csr):
+    # type: (Tensor, Tensor, str, bool) -> Tensor
     pass
 
 
-def spmm(src: Adj, other: Tensor, reduce: str = "sum") -> Tensor:
+def spmm(src: Adj, other: Tensor, reduce: str = "sum",
+         convert_to_csr: bool = False) -> Tensor:
```

Comment:
Do we desperately need this argument? I assume PyTorch will convert to CSR internally anyway?

Reply:
I will remove it and make the conversion obligatory. I thought that it might have been better for the user to control this, but you're right it - won't be necessary.